js/coinbase.js

```diff
@@ -1138,22 +1138,29 @@ module.exports = class coinbase extends Exchange {
         const url = this.urls['api'] + fullPath;
         if (api === 'private') {
             this.checkRequiredCredentials ();
-            const nonce = this.nonce ().toString ();
-            let payload = '';
-            if (method !== 'GET') {
-                if (Object.keys (query).length) {
-                    body = this.json (query);
-                    payload = body;
+            if (this.token) {
+                headers = {
+                    'Authorization': `Bearer ${this.token}`,
```

Comment:
Realize the Travis build failed, probably have to change this line:
`'Authorization': `Bearer ' + this.token,`